src/libYARP_dev/include/yarp/dev/ICurrentControl.h

```diff
@@ -102,76 +102,6 @@ class YARP_dev_API yarp::dev::ICurrentControl
     * @return true/false on success/failure
     */
     virtual bool getRefCurrent(int m, double *curr) = 0;
```

Comment:
@randaz81 just to know (i don't want to remove this as i like it).
Is this method equivalent to the `getPidReference(VOCAB_MODE_CURRENT, ...)`
and similar methods?